shell/store/index.js

```diff
@@ -778,7 +778,6 @@ export const actions = {
     }
 
     res = await allHash(promises);
-    dispatch('i18n/init');
```

Comment:
No point in having it here if we need it for every page and we're already making that call elsewhere.